### PR TITLE
Update page classes so we can use newer version of PF-next

### PR DESCRIPTION
**Pull request by lgalis**
[Screencast from 02-14-2019 09:47:47 AM.zip](https://github.com/ManageIQ/approval-ui/files/2865508/Screencast.from.02-14-2019.09.47.47.AM.zip)

screenshot from 2019-02-14 09-56-19
screenshot from 2019-02-14 09-55-43
screenshot from 2019-02-14 09-55-23
screenshot from 2019-02-14 09-55-08

### CODE REVIEW

**Comment by lgalis** [comment]
@gmcculloug - please review/merge

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
-    "@data-driven-forms/pf4-component-mapper": "^0.4.2",
-    "@data-driven-forms/react-form-renderer": "^1.7.0",
+    "@data-driven-forms/pf4-component-mapper": "^0.4.1",
+    "@data-driven-forms/react-form-renderer": "^1.6.0",
```
Comment (Hyperkid123) [inline package.json]: Did you have some issues with newer version of the renderer?

File: src/App.scss [hunk 2]
```diff
@@ -34,3 +34,30 @@ ul.navigation-secondary {
...
+  align-items: baseline;
+}
+
+.modal-form-right-align {
+  .pf-m-action:last-child {
+    .data-driven-forms__pf4-button-group:first-child {
+      margin-left: auto
+    }
+  }
+}
```
Comment (Hyperkid123) [inline src/App.scss]: missing new line

File: package.json (hunk 1 above)
Comment (lgalis) [reply package.json]: No, I did not have any issues, updated to the new version

**Pull request merged**